### PR TITLE
Update CompanyFetch.js

### PR DESCRIPTION
**Pull request by shretimag**
- eliminated all the comments
- changed names of variables ( navigatecompanies, a and b used in sorting earlier, mapping variable for the code to be clear)
- removed axios component imported as it wasn't in use

### CODE REVIEW

File: src/components/InsightsComponents/CompanyFetch.js [hunk 1]
```diff
@@ -143,40 +69,55 @@ export default function CompanyFetch(props) {
...
+                  company_one.company_name.toLowerCase() >
+                  company_two.company_name.toLowerCase()
+                )
+                  return 1;
                 return 0;
-               })
-              .map((comp, idx) => (
-                < Card
+              })
+              .map((componies, idx) => (
```
Comment (Parth-kumar-1802) [inline src/components/InsightsComponents/CompanyFetch.js:96]: spell check

Comment (shretimag) [reply src/components/InsightsComponents/CompanyFetch.js:96]: sorry sir, on it

File: src/components/InsightsComponents/CompanyFetch.js [hunk 2]
```diff
@@ -22,119 +21,46 @@ const Container = styled.div`
...
-    let query=``;
-    if(props.year==2023){
-      query = `*[_type == "placement23"]{...}`
-    }
-    else{
-      query = `*[_type == "placement"]{...}`
+  useEffect(() => {
+    let query = ``;
+    if (props.year == 2023) {
+      query = `*[_type == "placement23"]{...}`;
```
Comment (Parth-kumar-1802) [inline src/components/InsightsComponents/CompanyFetch.js:33]: change "query" to better reflect what we are querying for...

File: src/components/InsightsComponents/CompanyFetch.js (hunk 1 above)
Comment (shretimag) [reply src/components/InsightsComponents/CompanyFetch.js]: done sir

File: src/components/InsightsComponents/CompanyFetch.js [hunk 3]
```diff
@@ -22,119 +21,46 @@ const Container = styled.div`
...
+    });
+  }, []);
+
   const year = props.year;
-  const yearFilter = companies.filter((element)=>{
+  const yearFilter = companies.filter((element) => {
     return element.year == year;
-  })
+  });
   console.log(yearFilter);
```
Comment (Parth-kumar-1802) [inline src/components/InsightsComponents/CompanyFetch.js:46]: not needed any more

**Pull request merged**